app/javascript/react/components/ReviewTile.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const ReviewTile = (props) => {
+    return (
+        <div>
+            <h4>
+                Rating: {props.rating} <br/>
+                Review: {props.body}
+            </h4>
+        </div>
```

Comment:
this div is not doing anything. Delete it! 

now that we are in projects, we should no longer ever use `br` tags. Instead use CSS to get the intended spacing and alignment